rector.php

```diff
@@ -68,6 +68,7 @@
         __DIR__ . '/phpstan.neon.dist',
         __DIR__ . '/vendor/codeigniter/phpstan-codeigniter/extension.neon',
         __DIR__ . '/vendor/phpstan/phpstan-strict-rules/rules.neon',
+        __DIR__ . '/vendor/phpstan/phpstan/conf/bleedingEdge.neon',
```

Comment:
you need to use real location of `bleedingEdge.neon` config under phpstan.phar, I will check when back to PC

Reply:
The phar location include a phar from relative path, follow include target path

Reply:
@paulbalandan you can use this path:

```
'phar://' . __DIR__ . '/vendor/phpstan/phpstan/phpstan.phar/conf/bleedingEdge.neon'
```